ray-operator/DEVELOPMENT.md

```diff
@@ -7,15 +7,27 @@ This section walks through how to build and test the operator in a running Kuber
 software  | version | link
 :-------------  | :---------------:| -------------:
 kubectl |  v1.21.0+    | [download](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-go  | v1.16,17|[download](https://golang.org/dl/)
+go  | v1.17|[download](https://golang.org/dl/)
 docker   | 19.03+|[download](https://docs.docker.com/install/)
 
-The instructions assume you have access to a running Kubernetes cluster via ``kubectl``. If you want to test locally, consider using [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+The instructions assume you have access to a running Kubernetes cluster via ``kubectl``. If you want to test locally, consider using [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/). 
 
 ### Setup on Kind
 
 For a local [kind](https://kind.sigs.k8s.io/) environment setup, you can follow the Jupyter Notebook example: [KubeRay-on-kind](../docs/notebook/kuberay-on-kind.ipynb).
 
+### Use go v1.17
+
+Currently, Kuberay does not support go v1.16 ([#568](https://github.com/ray-project/kuberay/issues/568)) and go v1.18 ([#518](https://github.com/ray-project/kuberay/issues/518)).
```

Comment:
Grammar nit:
`and` -> `or` because there's a negation (not)

It's as if English grammar follows De Morgan's law... but actually English grammar is just plain bizarre.

Reply:
Cool, I just wrote the sentence with De Morgan's law lol. Thank you for letting me know. Updated.